develop/plone/searching_and_indexing/query.rst

```diff
@@ -650,7 +651,7 @@ To get all catalog brains of certain content type on the whole site::
 
         campaign_brains = self.context.portal_catalog(portal_type="News Item")
 
-To see available type names, visit in portal_types tool in ZMI.
+To see available type names, visit in portal_types tool in Management Interface.
```

Comment:
remove the first "in" and "tool", so "visit portal_types in the Management Interface"